sprint2/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content=""width=device-widht , initial-scale="1.0" />
+    <link rel="stylesheet"; type="text/css"; href="style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" type="text/css"  href="style.css">
```
nu e nevoie de punct si virgula aici